honeybee_radiance_command/pcond.py

```diff
@@ -68,15 +68,16 @@ def to_radiance(self, stdin_input=False):
         command_parts = [self.command, self.options.to_radiance()]
         cmd = ' '.join(command_parts)
         if not stdin_input and self.input:
-            cmd = ' < '.join((cmd, self.input))
+            cmd = ' '.join((cmd, self.input))
```

Comment:
for two parameters like this probably styling the text is easier to read.

```suggestion
            cmd = '%s %s' % (cmd, self.input)
```